webapp/packages/plugin-projects/src/FolderDialog.tsx

```diff
@@ -120,7 +120,9 @@ export const FolderDialog: DialogComponent<FolderDialogPayload, IFolderDialogRes
   });
 
   function resolveHandler() {
-    resolveDialog({ folder: state.value, projectId: state.projectId });
+    if (state.valid) {
+      resolveDialog({ folder: state.value, projectId: state.projectId });
+    }
```

Comment:
please execute `await state.validate()` before to ensure that form is validate (usually it validated, but it's better to ensure it)